notebooks/py/DARTFISH.py

```diff
@@ -139,60 +154,25 @@ def compute_magnitudes(stack, norm_order=2):
 # EPY: END code
 
 # EPY: START markdown
-#### QC Plots
-# EPY: END markdown
-
-# EPY: START markdown
-##### parameter tuning plots
+### Compare to benchmark results
+#
+#The below plot aggregates gene copy number across single cells in the field of view and compares the results to the same copy numbers from the Matlab based pipeline. This can probably be improved by tweaking any of the above parameters.
```

Comment:
```suggestion
#The below plot aggregates gene copy number across cells in the field of view and compares the results to the same copy numbers from the authors' pipeline. This can likely be improved by tweaking parameters in the above algorithms.
```